torch/nn/functional.py

```diff
@@ -2773,19 +2774,6 @@ def gaussian_nll_loss(
     r"""Gaussian negative log likelihood loss.
 
     See :class:`~torch.nn.GaussianNLLLoss` for details.
-
```

Comment:
I would keep this.

While we do like to reduce document redundancy, in a more perfect world modules would just wrap functions, and the functions would be documented.